pkg/rancher-desktop/router.ts

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Still don't have a full picture of what's going on,  but how does this code interact with the dynamic routes that are still described in `pkg/RD/layouts/defaults.vue` ?  Like what does this code give us that we don't get already from `layouts/default.vue` ? And do we need to account for the routes that are added at runtime?

Reply:
Unfortunately, `@rancher/shell` doesn't support convention-based routing that nuxt provides us while working with plugins. This means that we will have to transition over to maintaining our own router file in order to run Rancher Desktop properly as a plugin.